tests/phpunit/tests/admin/wpTermsListTable.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * @group admin
+ *
+ * @covers WP_Terms_List_Table
+ */
+class Tests_Admin_WpTermsListTable extends WP_UnitTestCase {
+
+	/**
+	 * List table.
+	 *
+	 * @var WP_Terms_List_Table $terms_list_table
+	 */
+	private $terms_list_table;
+
+	private static $admin_id;
+	private static $author_id;
+	private static $term_object;
+
+	const CATEGORY_TAXONOMY = 'category';
+
+	public static function set_up_before_class() {
+		parent::set_up_before_class();
+
+		self::$admin_id  = self::factory()->user->create( array( 'role' => 'administrator' ) );
+		self::$author_id = self::factory()->user->create( array( 'role' => 'author' ) );
+
+		self::$term_object = self::factory()->term->create_and_get( array( 'taxonomy' => self::CATEGORY_TAXONOMY ) );
+
+		require_once ABSPATH . 'wp-admin/includes/class-wp-list-table.php';
+		require_once ABSPATH . 'wp-admin/includes/class-wp-terms-list-table.php';
+	}
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->terms_list_table = new WP_Terms_List_Table();
+	}
+
+	/**
+	 * Call a private method as if it was public.
+	 *
+	 * @param object|string $object      Object instance or class string to call the method of.
+	 * @param string        $method_name Name of the method to call.
+	 * @param array         $args        Optional. Array of arguments to pass to the method.
+	 * @return mixed Return value of the method call.
+	 * @throws ReflectionException If the object could not be reflected upon.
+	 */
+	private function call_private_method( $object, $method_name, $args = array() ) {
```

Comment:
```suggestion
	private function call_inaccessible_method( $object, $method_name, $args = array() ) {
```

Reason: the function you are calling is `protected`, not `private`, so the suggested new name is more appropriate.